ArduPlane/ArduPlane.cpp

```diff
@@ -245,7 +249,12 @@ void Plane::update_logging2(void)
 void Plane::afs_fs_check(void)
 {
     // perform AFS failsafe checks
-    afs.check(failsafe.last_heartbeat_ms, geofence_breached(), failsafe.AFS_last_valid_rc_ms);
+#if AC_FENCE
```

Comment:
By convention we always compare this against `== ENABLED`.  It was like that when I got here.

Reply:
Ack - of all the places I added that check, I was bound to miss one.